x-pack/plugins/fleet/server/services/telemetry/fleet_usage_sender.ts

```diff
@@ -126,6 +128,13 @@ export class FleetUsageSender {
       upgradeDetails.forEach((upgradeDetailsObj) => {
         core.analytics.reportEvent(FLEET_AGENTS_EVENT_TYPE, { upgrade_details: upgradeDetailsObj });
       });
+
+      appContextService
+        .getLogger()
+        .debug(() => 'Integrations details telemetry: ' + JSON.stringify(integrationsDetails));
+      core.analytics.reportEvent(FLEET_INTEGRATIONS_EVENT_TYPE, {
```

Comment:
we should register the new type with a schema like it's done here: https://github.com/elastic/kibana/blob/main/x-pack/plugins/fleet/server/services/telemetry/fleet_usage_sender.ts#L173-L182

it would be best to create a new schema object (and we should eventually separate the fleet_agents schema from fleet_usages)